crates/biome_cli/src/execute/traverse.rs

```diff
@@ -657,19 +667,22 @@ impl<'ctx, 'app> TraversalContext for TraversalOptions<'ctx, 'app> {
         }
     }
 
-    fn handle_path(&self, path: &Path) {
+    fn handle_path(&self, path: &BiomePath) {
         handle_file(self, path)
     }
 
-    fn store_path(&self, path: &Path) {
-        self.evaluated_paths.write().unwrap().insert(path.into());
+    fn store_path(&self, path: &BiomePath) {
+        self.evaluated_paths
+            .write()
+            .unwrap()
+            .insert(BiomePath::new(path.as_path()));
```

Comment:
Generally it's considered a bit of an antipattern in Rust to clone data inside a method that was passed by reference. If you need to clone, it's preferred to let the caller do it, since sometimes they may be able to avoid the clone by moving instead.

This method then becomes:

```suggestion
    fn store_path(&self, path: BiomePath) {
        self.evaluated_paths.write().unwrap().insert(path);
```